grails-app/migrations/0.8.x/changelog-2021-12-14-1140-add-submit-request-activity.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+  <changeSet author="MagdaToczek" id="1412202111400-0">
```

Comment:
Can you add a comment to this changeset to explain what's going on, including information about what location type you're targeting (location_type_id = 2)? 

Also we should be using location_type_code = DEPOT instead. But I suspect that makes the insert more difficult, so I think we can leave this as-is.

Reply:
ok

Reply:
@jmiranda why do you think the insert will be more difficult in this case. What if someone changed the id of the location type with code `DEPOT`? I think we can not rely on our inner ids too much.

Reply:
As my excuse I followed on existing migration (changelog-2018-07-09-2222-insert-internal-receiving-location-type.xml)